dbms/src/AggregateFunctions/AggregateFunctionGroupBitmapData.h

```diff
@@ -1,11 +1,15 @@
 #pragma once
 
 #include <algorithm>
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wcast-qual"
+#pragma GCC diagnostic ignored "-Wold-style-cast"
```

Comment:
We can include these headers as SYSTEM in CMake. This will imply `-isystem` for compiler option and subsequently, it means "ignore all warnings in this header".